### PR TITLE
Guest user

### PR DESCRIPTION
**Pull request by aryan-karadia**
1. Added guest user GUI implementation
2. Added Comfort seat options
3. Updated seat chart to now show seats as seen below
4. Added checking to see if a seat has already been booked using the 'orders' table, also seen below (E16)

image

### CODE REVIEW

**Comment by aryan-karadia** [comment]
1. Added Order table update when seat is booked
2. Added New fields when guest is buying seat
3. Added new Register button in guest welcome panel
4. Added navigability for each Frame
5. Added new frame closing for cleaner user experience

File: src/LoginFrame.java [hunk 1]
```diff
@@ -832,23 +1053,30 @@ public ConfirmationFrame(String username, Flight selectedFlight, boolean isEcono
...
                 LocalDate arrivalDate, LocalTime arrivalTime) {
             setTitle("Booking Confirmation");
-            setSize(300, 200); // Adjust the size as needed
+            setSize(500, 300); // Adjust the size as needed
             setLayout(new BorderLayout());
 
+            int OrderID = updateDatabase(username, selectedFlight.getFlightID(),
+                    selectedFlight.getAircraft().getAircraftModel(), selectedFlight.getDepartureLocation(),
+                    selectedFlight.getArrivalLocation(), departureTime, arrivalTime,(isEconomy ? "Economy" : isBusiness
+                            ? "Business" : "Comfort"), seatNumber, hasInsurance, totalPrice);
```
Comment (aaron-dalbroi) [inline src/LoginFrame.java]: On line 1053, I see double totalprice. On line 1062 its totalPrice. This was causing errrors

File: src/LoginFrame.java [hunk 2]
```diff
@@ -832,23 +1053,30 @@ public ConfirmationFrame(String username, Flight selectedFlight, boolean isEcono
...
             infoPanel.add(new JLabel("From: " + selectedFlight.getDepartureLocation()));
             infoPanel.add(new JLabel("To: " + selectedFlight.getArrivalLocation()));
-
             infoPanel.add(new JLabel("Departure time: " + departureTime));
             infoPanel.add(new JLabel("Arrival time: " + arrivalTime));
             infoPanel.add(new JLabel("Class: " + (isEconomy ? "Economy" : isBusiness ? "Business" : "Comfort")));
             infoPanel.add(new JLabel("Seat: " + seatNumber));
             infoPanel.add(new JLabel("Insurance: " + (hasInsurance ? "Yes" : "No")));
-            infoPanel.add(new JLabel("total price: " + totalprice));
+            infoPanel.add(new JLabel("total price: " + totalPrice));
```
Comment (aaron-dalbroi) [inline src/LoginFrame.java:1225]: Same issue as above

File: src/LoginFrame.java [hunk 3]
```diff
@@ -886,6 +1113,71 @@ public ConfirmationFrame(String username, Flight selectedFlight, boolean isEcono
                 JOptionPane.showMessageDialog(this, "Error updating database: " + ex.getMessage(),
                         "Database Error", JOptionPane.ERROR_MESSAGE);
```
Comment (aaron-dalbroi) [inline src/LoginFrame.java]: The try block at 1103 is not closed. Two curly braces fixed it for me.

**Review by aaron-dalbroi — COMMENTED** [review]
Heres where the errors that prevented me from compiling are

**Pull request merged**